content/python/concepts/math-module/terms/math-floor/math-floor.md

```diff
@@ -0,0 +1,49 @@
+---
+Title: 'math.floor()'
+Description: 'Takes in a numeric data type and rounds the value down to the nearest integer.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+Takes in a numeric data type and rounds the value down to the nearest integer.
+
+## Syntax
+
+```pseudo
+math.floor(n)
+```
+
+The **`math.floor()`** function takes in a value and returns its floor, the smallest integer greater than or equal to that value.
+
+## Example
+
+Use `math.floor()` to return the floor of `134.2`:
+
+```py
+import math
+
+print(math.floor(134.2))
+
+# Output: 134
+```
+
+## Codebyte Example
+
+The following example is runnable and uses the `math.floor()` function to return the floor of a number:
+
+```codebyte/python
+import math
+
+print(math.floor(4.2))
+# Output: 4
+
+print(math.floor(9.7))
+# Output: 9
```

Comment:
```suggestion
import math

print(math.floor(4.2))

print(math.floor(9.7))
```
We want to get the users to run the codeblock and find the output themselves so no need to give the output.